project_test.go

```diff
@@ -771,7 +771,7 @@ func TestProjectInfo(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	table := cmd.Table{Headers: cmd.Row{"Environment", "Address", "Version", "Deploy date"}}
+	table := cmd.Table{Headers: cmd.Row([]string{"Environment", "Address", "Version", "Deploy date"})}
```

Comment:
I'm curious, what is the difference between both statements? And what is the bug exactly?